modules/tinymce/src/core/main/ts/api/commands/NewlineCommands.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import * as InsertBr from '../../newline/InsertBr';
+import * as InsertNewLine from '../../newline/InsertNewLine';
+import type Editor from '../Editor';
+
+export const registerCommands = (editor: Editor) => {
```

Comment:
```suggestion
export const registerCommands = (editor: Editor): void => {
```

Reply:
Why would we need to specify the void type here it's inferred? If we need to there is a lot of places it needs to be added. It will be quite verbose if we added them to every function for every command?

Reply:
I explained this here: https://github.com/tinymce/tinymce/pull/7572#discussion_r795337514. The intention of that rule is to make sure you cannot accidentally break a module or public API by making internal changes and has helped out before. As such, that rule requires an explicit return type on exported functions only, which is where I've left the comments. I also don't want us adding more things that would add to the work to get that enabled, as I've already been working slowly to get all issues fixed.

Reply:
But this isn't exported. It just called internally to register things. Or does the rule apply to anything inside the api directory exposed or not?

Reply:
It's a TS lint rule, not a Tiny/Ephox rule so it applies to at the module level.